internal/db/dbutil/dbutil.go

```diff
@@ -153,6 +163,15 @@ func MigrateDB(db *sql.DB, dataSource string) error {
 	return err
 }
 
+type stdoutLogger struct{}
+
+func (stdoutLogger) Printf(format string, v ...interface{}) {
+	fmt.Printf(format, v...)
+}
+func (logger stdoutLogger) Verbose() bool {
```

Comment:
```suggestion
func (stdoutLogger) Verbose() bool {
```

It's also stylistically fine to make short functions single line, if you want to save some vertical space.